contracts/EverlongStrategyKeeper.sol

```diff
@@ -338,6 +338,12 @@ contract EverlongStrategyKeeper is Ownable {
             return 0;
         }
 
+        // If the strategy is a wrapped token, convert the spend amount to
+        // be denominated in hyperdrive's token.
```

Comment:
This comment is a bit confusing to me. Something like the following could be clearer since Hyperdrive doesn't have a token and actually has a base and vault shares token:

```
If the strategy's asset is a wrapped token, convert the spend amount to be denominated in units of vault shares.
```

Having said that, I'm not sure I understand what this code is doing. `canSpend` is initially set to:

```
uint256 canSpend = IERC20(IEverlongStrategy(_strategy).asset())
            .balanceOf(_strategy);
```

Isn't this already in units of the wrapped token since Everlong is holding something like wstETH in this case?

Reply:
You're totally correct! This can go :)